src/frontend/overlay/src/components/molecules/info/ContestantViewLine.tsx

```diff
@@ -49,16 +49,19 @@ const TaskRow = styled.div<TaskRowProps>`
 
   width: 100%;
 
-  border-radius: ${props => props.index === 0 && props.roundTop ? "16px" : 0} ${props => props.index === props.problems - 1 && props.roundTop ? "16px" : 0}  ${props => props.index === props.problems - 1 && props.roundBottom ? "16px" : 0} ${props => props.index === 0 && props.roundBottom ? "16px" : 0};
+  border-radius: ${props => props.index === 0 && props.roundTop ? c.GLOBAL_BORDER_RADIUS : 0} 
+      ${props => props.index === props.problems - 1 && props.roundTop ? c.GLOBAL_BORDER_RADIUS : 0}
```

Comment:
Maybe don't pass problems to props, and instead pass isFirstRow isLastRow?  
Or better use :first-child :last-child pseudoselectors